### ISSUE TITLE
0.3.0 File Creation Issues

### ISSUE THREAD

**Issue by bluesam3** [olivierkes/manuskript#37]
OK, we've got a couple, that I strongly suspect are related (Running Ubuntu 16.04 LTS, new install, nothing significant changed):

1) Saves aren't working: single-file saves randomly fail to open, the new open format files don't produce a .msk file, and so can't be opened at all. 

2) Settings are completely forgotten between sessions.

**Comment by olivierkes** [comment]
Any error message in the console?

**Comment by bluesam3** [comment]
On loading/saving the sample project and a test project:

Warning: failed to load translator for locale en_GB...
Loading: /home/sam/programs/writing/manuskript-0.3.0-linux-1/test.msk
Detected file format version: 1. Zip: False.
Project /home/sam/programs/writing/manuskript-0.3.0-linux-1/test.msk loaded.
Loading: /home/sam/programs/writing/manuskript-0.3.0-linux-1/test.msk
Detected file format version: 1. Zip: False.
Project /home/sam/programs/writing/manuskript-0.3.0-linux-1/test.msk loaded.
Loading: /home/sam/programs/writing/manuskript-0.3.0-linux-1/sample-projects/book-of-acts.msk
Detected file format version: 1. Zip: False.
Project /home/sam/programs/writing/manuskript-0.3.0-linux-1/sample-projects/book-of-acts.msk loaded.
Project /home/sam/programs/writing/manuskript-0.3.0-linux-1/sample-projects/book-of-acts.msk saved.
Traceback (most recent call last):
  File "manuskript/ui/welcome.py", line 119, in loadRecentFile
  File "manuskript/mainWindow.py", line 311, in loadProject
  File "manuskript/mainWindow.py", line 497, in loadDatas
  File "manuskript/loadSave.py", line 36, in loadProject
  File "zipfile.py", line 923, in __init__
IsADirectoryError: [Errno 21] Is a directory: '/home/sam/writing/2016bk1'
Traceback (most recent call last):
  File "manuskript/ui/welcome.py", line 133, in openFile
  File "manuskript/mainWindow.py", line 311, in loadProject
  File "manuskript/mainWindow.py", line 497, in loadDatas
  File "manuskript/loadSave.py", line 58, in loadProject
  File "/usr/lib/python3.4/codecs.py", line 319, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa4 in position 10: invalid start byte
^CTraceback (most recent call last):
  File "manuskript/ui/welcome.py", line 197, in changeTemplate
KeyboardInterrupt
Project /home/sam/programs/writing/manuskript-0.3.0-linux-1/test3.msk saved.
Project /home/sam/programs/writing/manuskript-0.3.0-linux-1/test3.msk saved.
Project /home/sam/programs/writing/manuskript-0.3.0-linux-1/test3.msk saved.
Loading: /home/sam/programs/writing/manuskript-0.3.0-linux-1/sample-projects/book-of-acts.msk
Detected file format version: 1. Zip: False.
Project /home/sam/programs/writing/manuskript-0.3.0-linux-1/sample-projects/book-of-acts.msk loaded.
Project /home/sam/programs/writing/manuskript-0.3.0-linux-1/sample-projects/book-of-acts.msk saved.
Loading: /home/sam/programs/writing/manuskript-0.3.0-linux-1/test3.msk
Detected file format version: 1. Zip: False.
There are some doublons: ['0', '0']
Project /home/sam/programs/writing/manuskript-0.3.0-linux-1/test3.msk loaded.

**Comment by olivierkes** [comment]
OK there are encoding issues, some of which should have been fixed.

Can you try the develop branch to see if those issues are resolved?

**Comment by mmacdonaldsilvercity** [comment]
This - the inability to save a project or file - is what would be considered a sev-1 level problem in a production shop. It is worth an emergency patch/promotion to the master.  I almost went down the path of debugging an already-solved issue, saved only by a check of the bug reports. Less technical folks will just walk away, even in light of it being early development. Beta or not, unless there is some really compelling reason to the contrary, this should be fixed on the master so it is not propagated any further.

**Comment by gedakc** [comment]
@olivierkes, it appears that the question you posed earlier was not answered:

> Can you try the develop branch to see if those issues are resolved?

I can confirm that the current development branch does indeed fix the issue.  I am now able to successfully create and save a new project.

**Comment by mmacdonaldsilvercity** [comment]
I did figure out a workaround for the time being, as I had no luck with the development install.  I may have mucked that up somehow, though... I'm on arch linux, and originally installed the master branch as per the AUR package build.  I uninstalled, then edited the PKGBUILD file to point to the development branch and reinstalled.  After that the product got even less along.  Reinstalled the master branch.
The workaround:  Save the demo project as my project, and edit to match my needs.
Outside of this bug, it looks really cool and I look forward to exercising it more.
